### ISSUE TITLE
Email null for Facebook, twitter, linkedin

### ISSUE THREAD

**Issue by rajeevjaiswal** [gorbin/ASNE#112]
Implemented this Library basic info fetching works but user email remains null for all the social logins mentioned.
@gorbin  @frostymarvelous will this be fixed or any workaround? Else I will have to implement above logins by normal way.

**Comment by frostymarvelous** [comment]
Are you using the appropriate permissions?

**Comment by rajeevjaiswal** [comment]
@frostymarvelous Yes for fb email is passed in fbscope and for twitter nothing is mentioned in the tutorial so kept it like that only as per turtorials still getting null value for email.
LinkedIn works perfectly.
I have implemented 

@Override
    public void onRequestDetailedSocialPersonSuccess(int socialNetworkId, SocialPerson socialPerson) {
    
        Log.d("Detail",""+socialPerson);
    }

To access full details data of user how can I access LinkedInPerson.class or Do I need to get it from socialPerson array

**Comment by usmanrana07** [comment]
any success regarding this issue?

**Comment by georgethomas2015e** [comment]
Please solve this issue its very important when someone using social login
